Json.More/JsonArrayTupleConverter.cs

```diff
@@ -78,26 +75,18 @@ public override (T1, T2) Read(ref Utf8JsonReader reader, Type typeToConvert, Jso
 	{
 		if (reader.TokenType != JsonTokenType.StartArray)
 			throw new JsonException("Expected tuple to be encoded as an array.");
-		reader.Read();
-
-		options = JsonCollectionTupleConverter.ManageConverters(options);
-
-		var value = options.Read<(T1, T2)>(ref reader);
-		reader.Read();
 
-		if (reader.TokenType != JsonTokenType.EndArray)
-			throw new JsonException("Expected end of array.");
-		reader.Read();
+		var array = JsonElement.ParseValue(ref reader);
```

Comment:
Why the need to allocate an intermediate JsonElement? Couldn't the value reader accept the `Utf8JsonReader` directly without making a copy?

Reply:
I suppose so, yeah. I had initially needed to know the size of the array, but it seems I've iterated away from that.

Reply:
Also, is it really an allocation if `JsonElement` and the enumerator are structs?

Reply:
So, it looks like because of the reflection, I can't use the reader directly because I need to pass it in by-ref, but I can't do that with `MethodInfo.Invoke()` because I can't put a by-ref into an object array.

If I can get around the reflection somehow, then I could do this, but I don't know how to get around it.

Once I hit more than 7 values, I have a `TRest` that I need to deal with.  I know that it's a `ValueTuple<...>`, but I don't know how many parameters there are, and that's where I need the reflection.  I can't do any static checks like `TRest is ValueType<...>` because I don't know what the `...` consists of or even how many type params there are.